### DISCUSSION TITLE
How to set the default selected value for a multi-select field.

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#9022] [Archive]
How to set the default selected value for a multi-select field.
Thanks

**Comment by benhaynes** [comment]
Put the option's key in the field's default value.

**Reply by cdwmhcc** [reply]
> Put the option's key in the field's default value.

![image](https://user-images.githubusercontent.com/8030134/138321411-afef33a0-c420-461c-9f34-07662c55b74a.png)

![image](https://user-images.githubusercontent.com/8030134/138321384-b21e0b13-d817-4b28-81b6-d1299e0b13a7.png)

When I set the default value, it cannot be selected：

![image](https://user-images.githubusercontent.com/8030134/138321319-d682a3fb-493f-4071-aafe-e98a53acdd07.png)

**Reply by benhaynes** [reply]
Hmm, not sure then. @rijkvanzanten ?

**Comment by joselcvarela** [comment]
Hello @cdwmhcc 
What database are you using?

I've tested here and it seems to work properly with postgres.
Also, in default value you should put the `value` configured in checkboxes, not the `text` .

**Reply by cdwmhcc** [reply]
mysql 8.0
